packages/compartment-mapper/test/fixtures-cjs-compat/node_modules/parser-struggles/main.js

```diff
@@ -0,0 +1,16 @@
+function nothing() {
+  const require = () => {};
+  require('spam', 'asifitsdoingsomething');
+  require('./spam');
+//   require('/spam'); // TODO: prevent this from causing errors 
```

Comment:
I mean to defer the error in a similar fashion to what I already did here. 
So that it happens, but in execution, not earlier. 

All I intend to prevent is a userspace function that happens to be called require tripping up loading. 
And I don't even know if that's a reasonable thing to fix yet, gotta find where the error originates from.